hello-strongbox-gradle/build.gradle

```diff
@@ -9,8 +13,35 @@ compileJava {
 
 description = "Example project that shows how to use gradle to build and deploy artifacts to Strongbox"
 
-ext.username = System.getProperty('credentials.username') != null ? System.getProperty('credentials.username') : mavenUser
-ext.password = System.getProperty('credentials.password') != null ? System.getProperty('credentials.password') : mavenPassword
+// publishing to GitHub Package Registry
+publishing {
+    repositories {
+        maven {
+            name = "GitHubPackages"
+            url = uri("https://maven.pkg.github.com/j4ckofalltrades/strongbox-examples")
```

Comment:
This would have to be changed before merging.

Reply:
Whoops, yes forgot that.